media-sound/supercollider/supercollider-3.8.0.ebuild

```diff
@@ -28,9 +28,14 @@ RDEPEND="
 	jack? ( media-sound/jack-audio-connection-kit )
 	portaudio? ( media-libs/portaudio )
 	qt5? (
+		dev-qt/linguist-tools:5
+		dev-qt/qtconcurrent:5
 		dev-qt/qtcore:5
+		dev-qt/qtdeclarative:5
```

Comment:
No, I did not know these tools exist (yet). I used a docker container stage3 to build it.